docs/usage/ipv6.md

```diff
@@ -50,19 +34,23 @@ When provisioning a DualStack cluster, the GCP provider creates distinct subnets
 
 - This subnet is dedicated to IPv6 services. It is created due to GCP's limitation of not supporting IPv6 reservation for services.
 
+### 3. **Internal Subnet (optional)**
+
+- This subnet is dedicated for internal load balancer. Currently are only internal IPv4 loadbalancer supported. They are provisioned by the Cloud Controller Manager (CCM).
+
 ---
 
 ## Additional Components
 
 ### 1. **Ingress-GCE**
 
-- The ingress-gce is a mandatory component for DualStack clusters. It is responsible for creating IPv6 Load Balancers. This is necessary because the GCP Cloud Controller Manager (CCM) does not support IPv6 Load Balancer creation.
+- The ingress-gce is a mandatory component for DualStack clusters. It is responsible for creating DualStack (IPv4,IPv6) Load Balancers. This is necessary because the GCP Cloud Controller Manager does not support provisioning IPv6 Load Balancer.
 
 ---
 
 ## Cloud Allocator (IPAM)
```

Comment:
Let's drop this whole section. It is not relevant for users as pod IP address management should just work.

If you want to keep it move it to a different document for extension developers.